.mergify.yml

```diff
@@ -29,7 +32,7 @@ pull_request_rules:
     actions:
         comment:
           message: |
-            This pull request is now in conflicts. Could you fix it @{{author}}? 🙏
+            This pull request is now in conflicts. Could you fix it? 🙏
             To fixup this pull request, you can check out it locally. See documentation: https://help.github.com/articles/checking-out-pull-requests-locally/
             ```
             git fetch upstream
```

Comment:
Wouldn't it be easier to ask the user to use a tool for this? Either running mergify locally or [backport](https://github.com/sqren/backport).

In Kibana we help the user like this:
![image](https://user-images.githubusercontent.com/209966/113850195-7cd4a300-979a-11eb-815a-95e1a3e6b7c5.png)

Reply:
good point. I don't know what's the tool the team have in place for that, in order to be be tool-agnostic the message provides the git commands to solve it locally. 

I'll keep a note to iterate on this and make the user experience smooth. Thanks for the feedback :)

Reply:
Most people use the `dev-tools/cherry-pick-pr` tool in the Beats repo it seems. But a few contributors use the nodejs backport tool. Both are supported. For the changelog we have a 'requirement' on the PR title to properly link to the original PR, so we can find unique PRs and double check if these features are really released (github tags can be missleading). Both tools are supported by our changelog scripts. Using custom git commands might be a problem if the title is not properly formatted (unless we add another lint check on the PR title for backports).

Reply:
We can then iterate with the existing tools, the current comment intends to help with fixing the just created backport PR that has got some conflicts, so we might need to think about that particular use case. I'll keep a note for the upcoming iterations